### PR TITLE
Adjust manifest sample for using module startupOrder

### PR DESCRIPTION
**Pull request by micahl**
Proposing these changes to version #'s used since my understanding is the feature was part of the 1.0.10 release (of edgeAgent, right?) and only recognized when using schemaVersion of at least 1.1.

### CODE REVIEW

File: doc/ModuleStartupOrder.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ The following sample deployment manifest illustrates how startupOrder values of
           "edgeAgent": {
             "type": "docker",
             "settings": {
-              "image": "mcr.microsoft.com/azureiotedge-agent:1.0",
+              "image": "mcr.microsoft.com/azureiotedge-agent:1.0.10",
```
Comment (yophilav) [inline doc/ModuleStartupOrder.md:61]: If the release process is done correctly, the docker image tag `1.0.10.1` and `1.0` should be synonymous. This change is uncessary.

Comment (yophilav) [inline doc/ModuleStartupOrder.md:61]: If the release process is done correctly, the docker image tag `1.0.10.1` and `1.0` should be synonymous. This change is unnecessary.

File: doc/ModuleStartupOrder.md [hunk 2]
```diff
@@ -116,7 +116,9 @@ The following sample deployment manifest illustrates how startupOrder values of
 
 In the sample deployment manifest shown above:
 
-* The `$edgeHub` module has been assigned a `startupOrder` value of 0.
+* The '$edgeAgent' schemaVersion has been set to 1.1 (or later).
+* The 'edgeAgent' module's image is 1.0.10 (or later).
```
Comment (yophilav) [inline doc/ModuleStartupOrder.md:120]: Per comment above, there is no need to update this image version.

File: doc/ModuleStartupOrder.md [hunk 3]
```diff
@@ -116,7 +116,9 @@ The following sample deployment manifest illustrates how startupOrder values of
...
+* The `edgeHub` module has been assigned a `startupOrder` value of 0.
```
Comment (yophilav) [inline doc/ModuleStartupOrder.md:121]: @richma-ms The `startupOrder` of `edgeAgent` is also 0, right? We can make an explicit comment here as well.

Comment (micahl) [reply doc/ModuleStartupOrder.md:121]: There's a note given in an earlier section of the doc describing the behavior for edgeAgent.  I'll reiterate it here as well.

File: doc/ModuleStartupOrder.md (hunk 2 above)
Comment (micahl) [reply doc/ModuleStartupOrder.md:120]: Ok, will remove.

File: doc/ModuleStartupOrder.md (hunk 1 above)
Comment (micahl) [reply doc/ModuleStartupOrder.md:61]: OK, removing this change.

**Review by yophilav — APPROVED** [review]

**Pull request merged**